### DISCUSSION TITLE
Creating transforms.json from own video/ set of images

### DISCUSSION THREAD

**Discussion by flicht** [NVlabs/instant-ngp#239] [Q&A]
Hi, 

How do I create a `transforms.json` file in the correct format to create a Nerf example from my own video or set of images? e.g. from a video from my iPhone. Without the GUI.

Thanks!

Awesome work :)

**Comment by Tom94** [comment]
https://github.com/NVlabs/instant-ngp/blob/master/docs/nerf_dataset_tips.md
